abtesting/Shared/AppConfig.swift

```diff
@@ -57,6 +57,31 @@ class AppConfig: ObservableObject {
     }
   }
 
+  @available(iOS 15, *)
+  func updateFromRemoteConfigAsync() async {
+    let remoteConfig = RemoteConfig.remoteConfig()
+    do {
```

Comment:
Is there a better way of dealing with the nested do-catch statements?

Reply:
No, this is just how error handling in Swift is.

Reply:
You could make the function throws, eliminate all the do, catches, and then do the do catch in the caller, but I think this is fine for this case other than my other comment.